### ISSUE TITLE
Trouble understanding load_and_play

### ISSUE THREAD

**Issue by BobbyBobBob-123** [monokim/PyRacing#2]
Hello @monokim  i am a student intrested in your work. I am quite new to python so i am having some trouble understanding how the lines under the 'def load_and_play' work  and how i can access this line of code as i have not seen this pop up ("Start loading history"). If you could explain to me how this works i would most appreciate it. Here is the code for reference:

def load_and_play():

    print("Start loading history")

    history_list = ['30000.npy']

    # load data from history file

    print("Start updating q_table")

    discount_factor = 0.99

    for list in history_list:

        history = load_data(list)

        learning_rate = get_learning_rate(0)

        print(list)

        file_size = len(history)

        print("file size : " + str(file_size))

        i = 0

        for data in history:

            state_0, action, reward, state, done = data

            best_q = np.amax(q_table[state])

            q_table[state_0 + (action,)] += learning_rate * (reward + discount_factor * (best_q) - q_table[state_0 + (action,)])

            if done == True:

                i += 1

                learning_rate = get_learning_rate(i)`

**Comment by monokim** [comment]
Hi Bobby, 
you can use that function if you have saved history(.npy)
It loads data for updating q_table (state, action, result, etc...)  from .npy file.
If you just want to start from the beginning, use simulate() function.

you can save 1 data pair(sate, action, result, etc...) into your array by using 
env.remember(state_0, action, reward, state, done)

and can save to file with
env.save_memory('filename')

**Comment by BobbyBobBob-123** [comment]
Hi monokim,
Thank you very much for responding.
